web/themes/contrib/civictheme/assets/sass/ckeditor/_ckeditor.scss

```diff
@@ -0,0 +1,15 @@
+//
+// This file contains styles for the CKEditor component in CivicTheme.
+//
+
+.ct-figure {
+  &.align-left,
+  &.align-center,
+  &.align-right {
+    display: block;
+    margin-block-start: ct-particle(2);
+    margin-block-end: ct-particle(2);
+    margin-inline-start: ct-particle-px(5);
+    margin-inline-end: ct-particle-px(5);
```

Comment:
I don't understand how these changes fix the issue.

Can you provide screenshots showing how this fixes the alignment issues mentioned in the ticket?

Reply:
I don't think this is quite working properly.

Looking at the screenshot, it looks like the "center" aligned image is really just left aligned, but sitting flush against the other image.

If you look at the screenshot below, the center would be slightly different if it was sitting properly in the center:

<img width="1114" height="731" alt="Screenshot 2025-07-29 at 1 51 18 pm" src="https://github.com/user-attachments/assets/24ec53d6-891d-4186-b348-2a6fa121dca1" />